packages/core/package.json

```diff
@@ -123,5 +128,6 @@
   "homepage": "https://github.com/mozilla-services/react-jsonschema-form#readme",
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "types": "index.d.ts"
```

Comment:
this is definitely an improvement, not sure what needs to be done to get them to remove this from @types as essentially this means that this project has taken ownership

Reply:
We will probably want to keep the @types typings to support people using the old package `react-jsonschema-form`. Since we've update the package name to `@rjsf/core` the typings will be included, so there will be no need to import `@types/react-jsonschema-form`.